src/modlunky2/ui/trackers/category.py

```diff
@@ -21,11 +21,85 @@
 ICON_PATH = BASE_DIR / "static/images"
 
 
+class CategoryModifiers(ttk.LabelFrame):
+    def __init__(self, parent, modlunky_config: Config, *args, **kwargs):
+        super().__init__(parent, *args, **kwargs)
+        self.parent = parent
+
+        self.modlunky_config = modlunky_config
+
+        self.always_show_modifiers = tk.BooleanVar()
+        self.always_show_modifiers.set(
+            modlunky_config.trackers.category.always_show_modifiers
+        )
+
+        widgets = []
+        widgets.append(
+            ttk.Checkbutton(
+                self,
+                text="Always Show",
```

Comment:
After some more thought, "Show all modifiers before 1-3" seems clearer? Now that the per-label checkboxes are in a row, there's room to be more verbose.

When I showed the UI sketch to my partner, this text was the part that confused them the most. The existing text "Always Show Modifiers" is leftover from the days of only supporting "No Gold" and "Pacifist" modifiers.